Sources/App/Settings/ActionConfigurator.swift

```diff
@@ -343,20 +334,7 @@ class ActionPreview: UIView {
     override func layoutSubviews() {
         super.layoutSubviews()
 
-        layer.cornerRadius = 5.0
-
-        layer.cornerRadius = 2.0
-        layer.borderWidth = 1.0
-        layer.borderColor = UIColor.clear.cgColor
-        layer.masksToBounds = true
-
-        layer.shadowColor = UIColor.black.cgColor
-        layer.shadowOffset = CGSize(width: 0, height: 2.0)
-        layer.shadowRadius = 2.0
-        layer.shadowOpacity = 0.5
-        layer.masksToBounds = false
-        layer.shadowPath = UIBezierPath(roundedRect: bounds, cornerRadius: layer.cornerRadius).cgPath
-
+        layer.cornerRadius = 8
```

Comment:
this is to match the (old) widget. if you want to make changes, go for it, but maybe update that or make it match the new widgets if you can

Reply:
I don't think it would make sense to update a feature that iOS itself is already is deprecating, I would focus on Apple Watch and new Widgets from now on, no need to remove the old "Today's view widget", but also not needed to take it in consideration when moving forward.

They already don't look that good
![Screenshot 2024-01-25 at 09 32 49](https://github.com/home-assistant/iOS/assets/5808343/6f218b72-12ed-420c-a0a0-f877e411b81c)